mkt/abuse/models.py

```diff
@@ -18,13 +19,15 @@ class AbuseReport(ModelBase):
     reporter = models.ForeignKey(UserProfile, null=True,
                                  blank=True, related_name='abuse_reported')
     ip_address = models.CharField(max_length=255, default='0.0.0.0')
-    # An abuse report can be for an addon, a user, or a website. Only one of
-    # these should be set.
+    # An abuse report can be for an app, a user, a website, or an extension.
+    # Only one of these should be set.
     addon = models.ForeignKey(Webapp, null=True, related_name='abuse_reports')
```

Comment:
I thought about changing it but there would be a likely cascade of addon->app changes needed...